app/serializers/rent_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class RentSerializer < ActiveModel::Serializer
+  attributes :id, :from, :to, :book, :user
```

Comment:
If you want the `book` and `user` to get serialized using BookSerializer and UserSerializer (99.9% of the time that's the case) you can't use attributes, you have to use something like:
```
class RentSerializer < ActiveModel::Serializer
  attributes :id, :from, :to
  belongs_to :book
  belongs_to :user
end
```

If you don't do it it'll just get all attributes and serialize them (totally bypassing the serializer)